src/Data/Traversable.js

```diff
@@ -0,0 +1,48 @@
+/* global exports */
+"use strict";
+
+// module Data.Traversable
+
+function Cont (fn) {
```

Comment:
I think putting them outside means a slight performance increase since it won't create a new closure every time. I'm not aware of any advantage to moving them inside.